src/themes/default/elements/emoji.variables

```diff
@@ -0,0 +1,20 @@
+/*******************************
+             Emoji
+*******************************/
+
+/*--------------
+    Path
+---------------*/
+@emojiPath: "https://cdn.jsdelivr.net/emojione/assets/4.0/png/128/";
```

Comment:
Load this as a local asset not via CDN

Take a look at icons for reference https://github.com/fomantic/Fomantic-UI/blob/master/LICENSE.md

Reply:
As far as i understood, it would be against their license to include the files in our own repository, but i cannot find the terms about this anymore atm. Anyhow: Should we really include those tons of files in the repo instead of letting the user download the icons on his ownfrom their website when wanted, and adjust the path to local assets on his own then?

Reply:
Yeah your right (https://www.emojione.com/licenses/free) this could be a problem